ldp/nn/agent/simple_local_agent.py

```diff
@@ -0,0 +1,149 @@
+from typing import cast
+
+import torch
+import torch.distributed as dist
+from aviary.core import Message, Tool, ToolRequestMessage
+from aviary_internal.nn.chunking import TensorChunker
+
+from ldp.agent import Agent, SimpleAgentState
+from ldp.graph import OpResult
+from ldp.graph.op_utils import compute_graph
+from ldp.llms import prepend_sys
+
+from ..graph.llm_call_op import LocalLLMCallOp  # noqa: TID252
+from ..handlers.transformer_handler import (  # noqa: TID252
+    ParallelModeConfig,
+    logits_to_logprobs,
+)
+from ..lm_config import LMConfig as _LMConfig  # noqa: TID252
+
+
+class LMConfig(_LMConfig):
+    """Adds some additional configuration options for running an LM in an Op."""
+
+    # async batching parameters
+    batch_size: int = 1
+    max_wait_interval: float = 0.1
+
+    # distribution
+    parallel_mode: ParallelModeConfig | None = None
```

Comment:
Can you add a `description` saying what happens if it's `None`?

Reply:
That's documented in `TransformerHandlerConfig`